### ISSUE TITLE
Audio plays two times on IE10 randomly

### ISSUE THREAD

**Issue by tazdev** [goldfire/howler.js#448]
A great library to work with.

I am facing the issue playing multiple sounds one after the other on internet explorer.

I have a single page application. On IE 10 often it happens that once the audio finishes the same audio plays again with next audio creating the chaos.

I have quite a few audios to be played in sequence so I am making a new Howl object for every audio.

I tried to figure out the issue and causes of it but couldn't.

```javascript
    var soundLoader = function(currPage, currIdx) {
      var baseRef = this;
      baseRef.currIdx = currIdx;
      baseRef.currPage = currPage;
      baseRef.idArr = currPage.audioArr;
      baseRef.loadedCount = 0;
      baseRef.soundObj = null;


      baseRef.loadCurrSound = function() {

        var audioID = baseRef.idArr[baseRef.loadedCount];
        var url = "assets/audio/" + audioID + ".mp3";
        baseRef.soundObj = new Howl({
          urls: [url],
          onload: baseRef.soundLoaded,
          loop: false,
          onplay: baseRef.onSoundPlay,
          onloaderror: baseRef.loadErrorHandler,
          onend: function() {
            onSoundCompleted(audioID);
          }
        });

        baseRef.soundObj.id = baseRef.idArr[baseRef.loadedCount];
      }


      baseRef.soundLoaded = function() {
        _audioCache.put(baseRef.soundObj.id, baseRef.soundObj);
        baseRef.loadedCount++;

        if (baseRef.loadedCount < baseRef.idArr.length) {
          baseRef.loadCurrSound();
        } else {
          // alert("### " + baseRef.soundObj.id)
          loadComplete(baseRef);
          //alert("done")
        }
      }

      baseRef.onSoundPlay = function() {

      }

      baseRef.loadCurrSound();
    }
```

**Comment by goldfire** [comment]
I tried testing this a few different ways using the 2.0 beta and wasn't able to reproduce it. I would suggest trying that out as there were lots of little issues like this that could only be solved by the 2.0 total rewrite.

**Comment by aroth** [comment]
@tazdev I am still seeing this in IE10 -- did you ever come up with a workaround?

**Comment by goldfire** [comment]
Are you referring to 2.0 or 1.x? I was never able to reproduce with 2.0.

**Comment by aroth** [comment]
Hi @goldfire, thanks for the quick reply.

I'm referring to the 2.0 branch. Tested on 2.0-beta8 and rc-1, IE10. The issue is difficult to reproduce because it can be quite intermittent. 

I've traced the issue down to #stop:

```
} else if (!isNaN(sound._node.duration) || sound._node.duration === Infinity) {
   sound._node.pause();
   sound._node.currentTime = sound._start || 0;
}
```

Using the following test  code:

```
var audio = new Howl({src:'/test.m4a', preload: false, html5: true}).play(); 
```

During my last two test runs, audio repeated 5 times out of 20 runs and then again 9 times out of 20. 

I suspect the issue is a bug/race condition in IE10's `Audio#pause` implementation. When setting `sound._node.currentTime` _after_ calling `sound._node.pause`,  the audio will occasionally repeat. 

However, if I flip these those two lines to become:

```
   sound._node.currentTime = sound._start || 0;
   sound._node.pause();
```

...audio no longer repeats. 

Any thoughts?
